roles/sap_install_media_detect/tasks/prepare/move_files_to_main_directory.yml

```diff
@@ -1,37 +1,40 @@
 ---
 
-- name: SAP Install Media Detect - Prepare - Initialize fact variables, phase 1b
+- name: SAP Install Media Detect - Prepare - Search known subdirectories - Initialize fact variables, phase 1b
```

Comment:
When running the role, prepare/create_file_list_phase_2.yml is expecting all files on the top level directory. However, the role is moving all SAP archives into subdirectories as part of running organize_files.yml. In order to achieve idempotency, we therefore need to move any existing files in lower level directories back to the top level directory. That's the purpose of the task in this file.

Because of this, please either remove the string `Search known subdirectories` from the task names of your PR, or modify it to something similar to `Move files back to top level`.

Why are we doing these file move operations?
- If we left the files on the top level after the role has finished, there would be many files of different SAP file types in one directory, which we wanted to avoid. Note: By setting `sap_install_media_detect_move_or_copy_archives` to `false`, the role can be configured to **not** move SAP archive files to subdirectories.
- If we searched for all files in lower level directories (except the extraction directories), there might be duplicates on the top level and we would need to handle this case as well.

Reply:
How about the verbose `- Known subdirectories search and move files to parent directory -` ?

Reply:
> How about the verbose `- Known subdirectories search and move files to parent directory -` ?

It would make the output lines very long and so the real task name could not be visible.

Reply:
That's why I went for something shorter, I'm not keen on just "Move files back to top level" because it has no context.

`- Move files to parent for known subdirs -` ?

Reply:
> `- Move files to parent for known subdirs -` ?

OK for me.